dags/test_dag.py

```diff
@@ -29,8 +29,7 @@
 }
 dag = DAG(DAG_NAME, schedule_interval='*/10 * * * *', default_args=default_args)
 
-run_this_1 = DummyOperator(task_id='run_this_1', dag=dag)
-run_this_2 = DummyOperator(task_id='run_this_2', dag=dag)
-run_this_2.set_upstream(run_this_1)
-run_this_3 = DummyOperator(task_id='run_this_3', dag=dag)
-run_this_3.set_upstream(run_this_2)
+run_this_1 = BashOperator(
```

Comment:
Why did you change this test?

Reply:
Just for a quick local test. Will be cleaned up when the approach has been validated or not :)